hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -1442,7 +1484,7 @@ def minimize_individual_nodes(self, chooser):
         attempt = self.cached_test_function_ir(lowered)
         if (
             attempt is None
-            or attempt.status < Status.VALID
+            or attempt.status < Status.INVALID
```

Comment:
I suppose this is reasonable? I didn't have a particularly principled reason for disallowing invalid results. Though this pass is explicitly aimed at the "size dependency on an integer draw" case, and I don't know how often reducing the integer draw in that case will result in an invalid result.

Reply:
Oh I think that can be reverted and isn't actually related to the rest of these changes and was something I was experimenting with, sorry. I'll remove it unless it breaks the tests (which I don't think it will)